packages/paste-website/src/components/shortcodes/PageAside.tsx

```diff
@@ -23,7 +22,7 @@ interface PageAsideProps {
 const PageAside: React.FC<PageAsideProps> = ({data}) => {
   return (
     <Box order={2} marginLeft="space140" minWidth="size20" id="page-aside" display={['none', 'none', 'block']}>
-      <Box position="sticky" top={STICKY_COLUMN_OFFSET}>
+      <Box position="sticky" top="space130">
```

Comment:
Was there a reason we had this as a constant before?

Reply:
Yeah, I think the way the flex boxes were set up before [the scroll fix pr](https://github.com/twilio-labs/paste/pull/1956)  - top had to account for the header/banner height otherwise it would go below them. Now, its structured better so the value doesn't need to do that.

Reply:
okay cool! thanks for explaining!